packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -15,6 +15,7 @@
   box-sizing: border-box;
   display: flex;
   position: relative;
+  border: 1px solid transparent;
```

Comment:
Border was used because that was what was asked for in the bug: https://github.com/OfficeDev/office-ui-fabric-react/issues/1387

Switching to outline...